metrics/config.go

```diff
@@ -41,8 +41,6 @@ const (
 	DomainEnv = "METRICS_DOMAIN"
 
 	// The following keys are used to configure metrics reporting.
-	// See https://github.com/knative/serving/blob/main/config/config-observability.yaml
```

Comment:
link is dead.

Reply:
The new link is https://github.com/knative/serving/blob/main/config/core/configmaps/observability.yaml, can we re-add it?